torch/linalg/__init__.py

```diff
@@ -82,6 +82,63 @@
     True
 """)
 
+inv = _add_docstr(_linalg.linalg_inv, r"""
+linalg.inv(input, *, out=None) -> Tensor
+
+This function computes the inverse matrix of a square matrix, or batch of such matrices, :attr:`input`.
+The result satisfies the relation
+``matmul(inv(input), input) = matmul(input, inv(input)) = eye(input.shape[0]).expand_as(input)``.
+
+Supports input of ``float``, ``double``, ``cfloat`` and ``cdouble`` data types.
+
+.. note:: If :attr:`input` is a non-invertible matrix or not a square matrix then a RuntimeError will be thrown.
```

Comment:
What about the batch case when one matrix is not invertible? 

Reminder: https://github.com/pytorch/pytorch/issues/25095, which will be fixed when we add inv_ex.

Reply:
Changed to 
```
.. note:: If :attr:`input` is a non-invertible matrix or non-square matrix, or batch with at least one such matrix,
          then a RuntimeError will be thrown.
```
This is tested here https://github.com/pytorch/pytorch/pull/48261/files#diff-7e17421f32124016eb8de04dc2f445da5786a28355e1addc72b305466f590180R2067